packages/runtime/src/next/client.ts

```diff
@@ -202,6 +208,32 @@ export class Makeswift {
     return json
   }
 
+  async getPage(
+    pathname: string,
+    { locale: localeInput }: { locale?: string } = {},
+  ): Promise<GetPageResult | null> {
+    const siteVersion = MakeswiftSiteVersion.Live
```

Comment:
Should we only enable this for Live version, or should we also enable this for the working version?

Reply:
I think that it should work for both. What are we supposed to render when we're looking at the working version of the site otherwise?

Reply:
Sure updated